actors/builtin/shared.go

```diff
@@ -43,6 +43,10 @@ type MinerAddrs struct {
 	Worker addr.Address
 }
 
+// This type is required to construct a miner (in power_actor) and change a miners multiaddress (in miner_actor)
+// the type is defined here to work around a circular dependency between the actors.
+type MultiAddress = []byte
```

Comment:
Why not just call it `Multiaddr`?

Reply:
No good reason, will call it that instead :+1: